### PR TITLE
Update teach-in instructions in supported.md

### PR DESCRIPTION
**Pull request by florianluediger**

### CODE REVIEW

File: doc/eep/supported.md [hunk 1]
```diff
@@ -127,11 +127,12 @@ __Pre-requirement__: [teach-in](#remote-device-send-data)
...
 
-You have to clear the old communication partner on the remote device and teach-in your system (see manual of the remote device, how to teach-in new devices).
+To perform a teach-in, you have to reset the receiver (eg. Smart-Plug) to clear all old communication partners. After that, you have to set your receiver into teach-in mode (read the manual of your device for instructions). Notice: Many devices have different teach-in modes, read your manual carefully. After a short time, your USB300/EnOcean-Pi modules(sender) should send a package to the receiver which causes the receiver to authorize the sender to send commands to the receiver.
 
-The most remote devices will be reply a teach-in request if a remote device sends a teach-in request.
+Technically the following happens:
+When in teach-in mode, the receiver will listen for all incoming telegrams. The Sender-ID of an arriving telegram (in this case sent from your sender module) is interpreted as an authorized information source. After that, every telegram with the authorized Sender-ID will be accepted and commands will be executed.
```
Comment (maggu2810) [inline doc/eep/supported.md]: The line you removed and the lines you added are a little bit different.
E.g. the FS1 smart plug device send a teach-in request and gets a response from the smarthome system.
Other devices that do not send packages and only listen for incoming ones handle a teach in as you describe.

**Comment by florianluediger** [comment]
Are there any chances, this PR gets merged?
Do you have any comments on my latest commit?

**Comment by maggu2810** [comment]
Sorry!

Github does not inform if new commits has been added to a branch / PR.
So I did never received a notification that you have changed the documentation after my remarks.
The best would be to add a "changes done" or "updates" comment, so the subscriber receives a message.

I will have a look at the next days.

**Comment by florianluediger** [comment]
Are there any updates?

Also remember this PR: aleon-GmbH/aleoncean-openhab-binding#18

**Comment by maggu2810** [comment]
Sorry about the delay and thank you again

**Pull request merged**